### PR TITLE
April Fool's Day:  Microwave Haywire Mode

### PR DESCRIPTION
**Pull request by Skyedra**
## About the PR
microwave

Now when you microwave metal, it makes the microwave go haywire!  The only way to stop it is destroy it.

**Media**

https://user-images.githubusercontent.com/22365940/228706626-e1bf1a97-124d-4e95-903e-7365643c3464.mp4

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Microwave goes haywire when you try to microwave metal.

### CODE REVIEW

File: Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs [hunk 1]
```diff
@@ -255,7 +262,34 @@ private void OnBreak(EntityUid uid, MicrowaveComponent component, BreakageEventA
...
+                    _audio.PlayPvs(component.FoodDoneSound, component.Owner, AudioParams.Default.WithVolume(-1));
+                    var random = new RobustRandom();
+
+                    // Make it look like the microwave exploded
+                    var machineFrame = Spawn("MachineFrameDestroyed", new Robust.Shared.Map.EntityCoordinates(uid, Vector2.Zero));
+                    Comp<TransformComponent>(machineFrame).Anchored = false;
+                    _throwingSystem.TryThrow(machineFrame, new Vector2(random.NextFloat()*20-10, random.NextFloat()*20-10), 10);
+
+                    // This doesn't seem to be spawning the microwave board, maybe someone else can figure out why?
+                    // It gets created and then instantly deleted, even with godmode and explosive resistance
```
Comment (moonheart08) [inline Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs:279]: You're spawning it as a child of the microwave.

Comment (Skyedra) [reply Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs:279]: Ahhhhh okay, I didn't realize Spawn worked like that, thanks!

Comment (Skyedra) [reply Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs]: Yeah, that totally fixed it, thanks! :D

**Comment by Skyedra** [comment]
Updated video that includes fix:  https://cdn.discordapp.com/attachments/310555209753690112/1090848873204953198/MicrowaveWithFix.mp4

**Pull request merged**